wrapper/src/main/java/software/amazon/jdbc/hostlistprovider/AuroraHostListProvider.java

```diff
@@ -93,6 +103,8 @@ public class AuroraHostListProvider implements DynamicHostListProvider {
   public static final CacheMap<String, String> suggestedPrimaryClusterIdCache = new CacheMap<>();
   public static final CacheMap<String, Boolean> primaryClusterIdCache = new CacheMap<>();
 
+  private static final int defaultTopologyQueryTimeoutMs = 5000;
+  private static final int setNetworkTimeoutMs = 10000;
```

Comment:
Is this a reasonable timeout value for waiting for setNetworkTimeout to complete?

Reply:
Setting timeout is a call to set a socket setting. It doesn't require any network activity like sending/receiving data. I'd expect that it takes few millis to set it but 10s is more than enough.

Reply:
Should I lower it to a different value then?